### PR TITLE
amp-consent-v2-validator change

### PR DESCRIPTION
**Pull request by zhouyx**
Second part of #20253, follow up to  #20284

### CODE REVIEW

File: extensions/amp-consent/0.1/test/validator-amp-consent-multi.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+</head>
+<body>
+Hello, world.
+<!-- Invalid: multiple amp-consent -->
+<amp-consent id='ABC' layout='nodisplay'>
+  <div id='ui1'>UI1
+    <button>Accept</button>
+  </div>
+</amp-consent>
+<!-- Invalid: multiple amp-consent with type -->
```
Comment (honeybadgerdontcare) [inline extensions/amp-consent/0.1/test/validator-amp-consent-multi.html]: drop the `with type` since that is what is multiple here.

Comment (zhouyx) [reply extensions/amp-consent/0.1/test/validator-amp-consent-multi.html]: done

**Comment by zhouyx** [comment]
Thank you @honeybadgerdontcare for helping me figure out the solution here!

File: extensions/amp-consent/validator-amp-consent.protoascii [hunk 2]
```diff
@@ -50,9 +50,29 @@ tags: {  # <amp-consent>
...
+  }
+}
+
+tags: {  # <amp-consent> (type)
+  html_format: AMP
+  tag_name: "AMP-CONSENT"
+  spec_name: "amp-consent [type]"
+  requires_extension: "amp-consent"
   unique: true
+  requires: "meta name=amp-consent-blocking"
```
Comment (janwinkler) [inline extensions/amp-consent/validator-amp-consent.protoascii:68]: doesnt seem to work. validator is reporting error although zhouyx <meta name=amp-consent-blocking ...> is in the page

Comment (zhouyx) [reply extensions/amp-consent/validator-amp-consent.protoascii:68]: @janwinkler could you share the page source? 

I've tried including `<meta name="amp-consent-blocking" content="amp-ad">` to the document, and it passed validation check.

Comment (janwinkler) [reply extensions/amp-consent/validator-amp-consent.protoascii:68]: it seems its not documented, hence i only tried <meta name="amp-consent-blocking" content="">. might be good to add this to the amp-consent documentation ;-)

Comment (micajuine-ho) [reply extensions/amp-consent/validator-amp-consent.protoascii:68]: @zhouyx Do you remember why we want amp-consent elements with `type` attributes to always have a meta tag?

Comment (zhouyx) [reply extensions/amp-consent/validator-amp-consent.protoascii:68]: https://github.com/ampproject/amphtml/issues/17742#issuecomment-512564655

**Pull request merged**